quarkchain/cluster/slave.py

```diff
@@ -902,6 +903,9 @@ def remove_cluster_peer_id(self, cluster_peer_id):
             await shard.create_peer_shard_connections(
                 self.cluster_peer_ids, self.master
             )
+            self.shard_subscription_managers[
```

Comment:
I don't like initializing subscription manager inside shard state, prefer moving this part to line 922

```
sub_manager = SubscriptionManager()
self.shard_subscription_managers[full_shard_id] = sub_manager
shard = Shard(self.env, full_shard_id, self, sub_manager)
new_shards.append(shard)
```

Reply:
a second thought, current form is fine